controllers/ansibleee_controller.go

```diff
@@ -65,7 +65,7 @@ func (r *AnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	//log := r.Log.WithValues("ansibleee", req.NamespacedName)
 
 	instance, err := r.getAnsibleeeInstance(ctx, req)
-	if err != nil {
+	if err != nil || instance.Name == "" {
```

Comment:
Thanks, this stops it from crashing with the error below in case the CRD isn't created before it's started.

```
1.667488498502694e+09	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "AnsibleEE.redhat.com", "error": "no matches for kind \"AnsibleEE\" in version \"redhat.com/v1alpha1\""}
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start.func1.1
	/home/fultonj/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.13.0/pkg/source/source.go:139
```